src/IceRpc.Templates/Templates/IceRpc-Slice-Client/.template.config/template.json

```diff
@@ -1,57 +1,147 @@
 {
-    "$schema":"http://json.schemastore.org/template",
-    "author":"ZeroC",
-    "classifications":[
+    "$schema": "http://json.schemastore.org/template",
+    "author": "ZeroC",
+    "classifications": [
         "IceRPC",
         "Client",
         "Slice"
     ],
-    "identity":"IceRpc.Slice.Client.0.1",
-    "name":"IceRPC + Slice Client",
-    "description":"A project template for creating an IceRPC + Slice client console application.",
-    "shortName":"icerpc-slice-client",
-    "tags":{
-        "language":"C#",
-        "type":"project"
+    "identity": "IceRpc.Slice.Client.0.1",
+    "name": "IceRPC + Slice Client",
+    "description": "A project template for creating an IceRPC + Slice client console application.",
+    "shortName": "icerpc-slice-client",
+    "tags": {
+        "language": "C#",
+        "type": "project"
     },
-    "sourceName":"IceRpc-Slice-Client",
-    "preferNameDirectory":true,
-    "defaultName":"Client",
-    "symbols":{
-        "skipRestore":{
-            "type":"parameter",
-            "datatype":"bool",
-            "description":"If specified, skips the automatic restore of the project on create.",
-            "defaultValue":"false"
+    "sourceName": "IceRpc-Slice-Client",
+    "preferNameDirectory": true,
+    "defaultName": "Client",
+    "symbols": {
+        "skipRestore": {
+            "type": "parameter",
+            "datatype": "bool",
+            "description": "If specified, skips the automatic restore of the project on create.",
+            "defaultValue": "false"
+        },
+        "transport": {
+            "type": "parameter",
+            "description": "The transport to use for IceRPC connections.",
+            "datatype": "choice",
+            "choices": [
+                {
+                    "choice": "quic",
+                    "description": "Use QUIC transport"
+                },
+                {
+                    "choice": "slic",
```

Comment:
Assuming we keep the transport choice, we should not expose / talk about Slic here (note the spelling for Slic).

The transport is actually tcp / TCP. See for example https://github.com/icerpc/icerpc-csharp/tree/main/examples/slice/TcpFallback. It's "TCP fallback" not "QUIC fallback". Slic is only of interest to advanced users.